charts/testkube/values.yaml

```diff
@@ -592,6 +592,12 @@ testkube-api:
   clusterName: ""
   ## enable endpoint to list testkube namespace secrets
   enableSecretsEndpoint: false
+  ## enable only specified executors with enabled flag
```

Comment:
Comments in testkube should use `# --` prefix so helm-docs correctly parses it

Reply:
Looks like it's usual to do it this way in this file - so I would prefer refactor to be in separate PR if needed - let's not mix contexts.